Makefile

```
@@ -19,6 +19,16 @@ test: | $(grpc_services_directory)
 	$(REBAR) ct
 	$(REBAR) dialyzer
 
+# This will run all ct exepct the lora suite that for now cannot be ran on MacOS
+test-mac: | $(grpc_services_directory)
+	$(REBAR) fmt --verbose --check rebar.config
+	$(REBAR) fmt --verbose --check "{src,include,test}/**/*.{hrl,erl,app.src}" --exclude-files "src/grpc/autogen/**/*"
+	$(REBAR) fmt --verbose --check "config/{test,sys}.{config,config.src}"
+	$(REBAR) xref
+	$(REBAR) eunit
+	$(REBAR) ct --suite=router_SUITE,router_channel_aws_SUITE,router_channel_azure_SUITE,router_channel_console_SUITE,router_channel_http_SUITE,router_channel_mqtt_SUITE,router_channel_no_channel_SUITE,router_console_api_SUITE,router_console_dc_tracker_SUITE,router_data_SUITE,router_decoder_SUITE,router_decoder_custom_sup_SUITE,router_decoder_custom_worker_SUITE,router_device_channels_worker_SUITE,router_device_devaddr_SUITE,router_device_routing_SUITE,router_device_worker_SUITE,router_discovery_SUITE,router_downlink_SUITE,router_grpc_SUITE,router_metrics_SUITE,router_sc_worker_SUITE,router_v8_SUITE,router_xor_filter_SUITE
```

That will be too easily overlooked and not updated in future.

How about omitting just the non-`_SUITE.erl` files and those without `_lorawan_` in the name with a bit of old school cartoon swearing:

```suggestion
	$(REBAR) ct --suite=$(cd test/ && ls -1N *_SUITE.erl | grep -v _lorawan_ | awk '{gsub(".erl$", ""); printf("%s,", $0)}')
```